src/components/SiteDetail.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import summary from '../apis/summary';
+class SiteDetail extends Component {
+  state = {
+    loadedSites: null
+  };
+
+  componentDidMount() {
+    this.loadSiteData();
+  }
+
+  componentDidUpdate() {
```

Comment:
One other thing! The wonderful thing about `componentDidUpdate` is that you can compare props with previous props. 

This means that you can slim the function of `loadSiteData` to just "loading site data", so that the check doesn't happen in `componentDidMount`. 

you can read more here -> https://reactjs.org/docs/react-component.html#componentdidupdate

Reply:
yes, I keep this in mind later on for refactoring as more data is pulled in. This will be changed.

Reply:
Good enough for me!